ui/src/components/SessionProvider.tsx

```diff
@@ -43,7 +44,8 @@ export default function SessionProvider({
       }
 
       try {
-        const self: IAuthOIDCInternal | IAuthOIDCInternal = await getAuthSelf();
+        const self: IAuthOIDCInternal | IAuthOIDCInternal | IAuthJWTInternal =
```

Comment:
```suggestion
        const self: IAuthOIDCInternal | IAuthGithubInternal | IAuthJWTInternal =
```
I think this might be a current bug